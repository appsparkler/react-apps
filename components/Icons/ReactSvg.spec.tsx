import { render } from "@testing-library/react";
import { ReactSvg } from "./ReactSvg";

describe("ReactSVG Snapshot Tests", () => {
  it("should match snapshot", () => {
    const { container } = render(<ReactSvg />);
    expect(container).toMatchSnapshot();
  });
});
