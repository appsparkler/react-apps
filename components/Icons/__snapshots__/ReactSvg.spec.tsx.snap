// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReactSVG Snapshot Tests should match snapshot 1`] = `
<div>
  <svg
    viewBox="-11.5 -10.23174 23 20.46348"
    xmlns="http://www.w3.org/2000/svg"
  >
    <title>
      React Logo
    </title>
    <circle
      cx="0"
      cy="0"
      fill="#61dafb"
      r="2.05"
    />
    <g
      fill="none"
      stroke="#61dafb"
      stroke-width="1"
    >
      <ellipse
        rx="11"
        ry="4.2"
      />
      <ellipse
        rx="11"
        ry="4.2"
        transform="rotate(60)"
      />
      <ellipse
        rx="11"
        ry="4.2"
        transform="rotate(120)"
      />
    </g>
  </svg>
</div>
`;
